packages/camera/camera_android_camerax/android/src/main/java/io/flutter/plugins/camerax/DeviceOrientationManager.java

```diff
@@ -265,57 +150,18 @@ PlatformChannel.DeviceOrientation getUIOrientation() {
   }
 
   /**
-   * Calculates the sensor orientation based on the supplied angle.
-   *
-   * <p>This method is visible for testing purposes only and should never be used outside this
-   * class.
-   *
-   * @param angle Orientation angle.
-   * @return The sensor orientation based on the supplied angle.
-   */
-  @VisibleForTesting
-  PlatformChannel.DeviceOrientation calculateSensorOrientation(int angle) {
-    final int tolerance = 45;
-    angle += tolerance;
-
-    // Orientation is 0 in the default orientation mode. This is portrait-mode for phones
-    // and landscape for tablets. We have to compensate for this by calculating the default
-    // orientation, and apply an offset accordingly.
-    int defaultDeviceOrientation = getDeviceDefaultOrientation();
-    if (defaultDeviceOrientation == Configuration.ORIENTATION_LANDSCAPE) {
-      angle += 90;
-    }
-    // Determine the orientation
-    angle = angle % 360;
-    return new PlatformChannel.DeviceOrientation[] {
-          PlatformChannel.DeviceOrientation.PORTRAIT_UP,
-          PlatformChannel.DeviceOrientation.LANDSCAPE_LEFT,
-          PlatformChannel.DeviceOrientation.PORTRAIT_DOWN,
-          PlatformChannel.DeviceOrientation.LANDSCAPE_RIGHT,
-        }
-        [angle / 90];
-  }
-
-  /**
-   * Gets the default orientation of the device.
+   * Gets default capture rotation for CameraX {@code UseCase}s.
    *
-   * <p>This method is visible for testing purposes only and should never be used outside this
-   * class.
+   * <p>See
+   * https://developer.android.com/reference/androidx/camera/core/ImageCapture#setTargetRotation(int),
+   * for instance.
    *
-   * @return The default orientation of the device.
+   * @return The rotation of the screen from its "natural" orientation; one of {@code
+   *     Surface.ROTATION_0}, {@code Surface.ROTATION_90}, {@code Surface.ROTATION_180}, {@code
+   *     Surface.ROTATION_270}
    */
-  @VisibleForTesting
-  int getDeviceDefaultOrientation() {
-    Configuration config = activity.getResources().getConfiguration();
-    int rotation = getDisplay().getRotation();
-    if (((rotation == Surface.ROTATION_0 || rotation == Surface.ROTATION_180)
-            && config.orientation == Configuration.ORIENTATION_LANDSCAPE)
-        || ((rotation == Surface.ROTATION_90 || rotation == Surface.ROTATION_270)
-            && config.orientation == Configuration.ORIENTATION_PORTRAIT)) {
-      return Configuration.ORIENTATION_LANDSCAPE;
-    } else {
-      return Configuration.ORIENTATION_PORTRAIT;
-    }
+  int getDefaultRotation() {
```

Comment:
nit: I believe this method is named `getDefaultRotation` because it is the rotation that CameraX use cases (at least ImageCapture) default to right? I think we might want to name it something else, though, because without that context from [the comment here](https://developer.android.com/reference/androidx/camera/core/ImageCapture#setTargetRotation(int)), it mostly gets used in places where it looks like we are getting a default rotation of the camera proxy, (i.e. `proxy.getDefaultRotation()`) which at first glance doesn't seem like something that would be a function of the current rotation of the device, but rather a persistent default rotation. 

Basically it just feels more like a `getCurrentRotation` than a `getDefaultRotation`, to me at least. Not a big deal, though.

Reply:
To address this, I renamed the wrapped method to `getDefaultDisplayRotation()` and left this since this really a part of the `DeviceOrientationManager` and would like to keep it consistent with `camera_android` for now.